snapshot/storage/bolt.go

```diff
@@ -291,6 +291,24 @@ func CommitActive(ctx context.Context, key, name string) (id string, err error)
 			return errors.Wrap(err, "failed to delete active")
 		}
 
+		if pbkt != nil {
+			k, _ := pbkt.Cursor().Seek(parentPrefixKey(ss.ID))
```

Comment:
Why is this check needed? Actives shouldn't have children, but even if they do, the ID does not change on commit.